lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeBootstrapConfigurationGenerator.java

```diff
@@ -16,13 +16,22 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 package org.apache.reef.runtime.multi.client;
 
-import org.apache.reef.tang.formats.ConfigurationModuleBuilder;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.annotations.DefaultImplementation;
 
 /**
- * A ConfigurationModule to bind arbitrary named parameters for runtimes.
+ * Defines a contract for a multi runtime bootstrap configuration.
+ * This generator adds configurations that are needed for the multi-runtime to run,
+ * such as classpath provider etc.
  */
-final class MultiRuntimeHelperConfiguration extends ConfigurationModuleBuilder {
+@DefaultImplementation(EmptyMultiRuntimeBootstrapConfigurationGeneratorImpl.class)
```

Comment:
I think it's best practice to always be explicit. This can help the people who want to develop as well.